app/src/main/AndroidManifest.xml

```diff
@@ -10,6 +10,7 @@
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
```

Comment:
**Reporter**: ANDROIDLINT
**Rule**: RtlEnabled
**Severity**: WARN
**File**: C:/Users/anisart/.jenkins/workspace/test3/app/src/main/AndroidManifest.xml L0

Using RTL attributes without enabling RTL support&#10;The project references RTL attributes, but does not explicitly enable or disable RTL support with &#96;android:supportsRtl&#96; in the manifest&#10;To enable right-to-left support, when running on API 17 and higher, you must set the &#96;android:supportsRtl&#96; attribute in the manifest &#96;&lt;application&gt;&#96; element.&amp;#xA;&amp;#xA;If you have started adding RTL attributes, but have not yet finished the migration, you can set the attribute to false to satisfy this lint check.


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1247460132>*